mne_bids/mne_bids.py

```diff
@@ -340,6 +355,12 @@ def _sidecar_json(raw, task, manufacturer, fname, kind,
         warn('No line frequency found, defaulting to 50 Hz')
         powerlinefrequency = 50
 
+    # determine whether any channels have to be ignored:
+    num_ignored = len([ch_name for ch_name in
+                       IGNORED_CHANNELS.get(manufacturer, list()) if
+                       ch_name in raw.ch_names])
+    # all ignored channels are trigger channels at the moment...
+
```

Comment:
the `num_ignored` and the `IGNORED_CHANNELS` imply that this is about more than just "stimulus" channels. So if in the future we want to ignore other channels as well, the present logic will have to be modified. Otherwise we will run into trouble with regards to line 383, where we deduct `num_ignored` from `n_stimchan`

However I don't expect to ignore channels beside STIMULUS, so I would accept this solution as it is for now

Reply:
yep, that's why I left this comment here. As you say it wouldn't be too hard to modify the logic, but if it isn't needed right now it would make it a bit more complex than it needs to be.